sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,1471 @@
+-- Query for fenix_derived.feature_usage_events_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+DECLARE start_date DATE DEFAULT "2021-01-01";
+
+DECLARE end_date DATE DEFAULT current_date;
+
+WITH dau_segments AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    COUNT(DISTINCT client_info.client_id) AS dau
+  FROM
+    `mozdata.fenix.events_unnested`
```

Comment:
I'd suggest in these query files we reference the default project used by the job (`moz-fx-data-shared-prod`), in this case just removing the project id will work.
```suggestion
    fenix.events_unnested
```